annotation_utils/misc.py

```diff
@@ -1,63 +1,69 @@
 # Various utility functions used to manipulate the pilecam annotations
```

Comment:
This file is mostly formatting, when I sent the PR to format the code I missed this file because it's in a different directory.  I only added the `my_device()` function here